client/my-sites/site-settings/form-general.jsx

```diff
@@ -725,7 +726,14 @@ export class SiteSettingsFormGeneral extends Component {
 					showButton
 					title={ translate( 'Privacy {{learnMoreLink/}}', {
 						components: {
-							learnMoreLink: <InlineSupportLink supportContext="privacy" showText={ false } />,
+							learnMoreLink: (
+								<InfoPopover position="bottom right">
+									{ translate( 'Control who can view your site. ' ) }
+									<InlineSupportLink showIcon={ false } supportContext="privacy">
+										{ translate( 'Learn more' ) }
```

Comment:
I agree with @wojtekn - I think we need the dot similar to other cases:

<img width="736" alt="Screenshot 2024-01-09 at 10 05 18 AM" src="https://github.com/Automattic/wp-calypso/assets/25575134/3ca726c7-f313-4430-8b48-9ce9f49e81e9">